juniper_official/routing/Ospf3NeighborStatsTable.yml

```diff
@@ -16,6 +16,30 @@ Ospf3NeighborStatsView:
     ospf-neighbor-state: ospf-neighbor-state
     instance-name: ../../ospf-instance-name
 
+Ospf3StatsTable:
```

Comment:
Can the output be large? do we require filtering using xslt?

Reply:
For ospf3 stats output can be large so xslt would be needed, but for ospf3 io-stats its at the protocol level so not needed.

Reply:
Will same xslt work for both the tables?

Reply:
Have added seperate yml files for the rules.